src/averageAges.js

```diff
@@ -55,7 +68,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const kids = people.filter(kid => (
+    onlyWithSon
+      ? people.find(mother => mother.name === kid.mother)
```

Comment:
you dont need element that FIND method returns. You just need to khow that it is. (boolean value)
Use SOME method.

fix in all cases